ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -611,6 +611,39 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
       PROVIDER_IMAGE: cluster-api-provider-agent
     workflow: assisted-baremetal-operator-capi
+- as: e2e-ai-operator-disconnected-capi
```

Comment:
nit: let's move those two jobs after ``e2e-ai-operator-ztp-capi-periodic`` so we can preserve the convention of a presubmit+periodic job pairs being together (currently it's splitting between ``e2e-ai-operator-ztp-capi`` and ``e2e-ai-operator-ztp-capi-periodic``)